### PR TITLE
chore: Add PRs to triage project

### PR DESCRIPTION
**Pull request by nzakas**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[x] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

Infrastructure

#### What changes did you make? (Give an overview)

Adds PRs to the triage project so we can manage issues and PRs in one place.

I also updated the triaging documentation to explicitly mention pull requests.

Related PRs:

* https://github.com/eslint/espree/pull/580
* https://github.com/eslint/eslintrc/pull/121
* https://github.com/eslint/eslint-visitor-keys/pull/54
* https://github.com/eslint/eslint-scope/pull/104
* https://github.com/eslint/generator-eslint/pull/160
* https://github.com/eslint/eslint.org/pull/456
* 
#### Is there anything you'd like reviewers to focus on?

Does the documentation make sense.

### CODE REVIEW

File: docs/src/maintain/manage-issues.md [hunk 1]
```diff
@@ -17,53 +17,54 @@ New issues are filed frequently, and how we respond to those issues directly aff
 1. **Not all requests are equal.** It's unlikely we'll be able to accommodate every request, so don't be afraid to say that something doesn't fit into the scope of the project or isn't practical. It's better to give such feedback if that's the case.
 1. **Close when appropriate.** Don't be afraid to close issues that you don't think will be done, or when it's become clear from the conversation that there's no further work to do. Issues can always be reopened if they are closed incorrectly, so feel free to close issues when appropriate. Just be sure to leave a comment explaining why the issue is being closed (if not closed by a commit).
 
-## Types of Issues
+## Types of Issues and Pull Requess
```
Comment (amareshsm) [inline docs/src/maintain/manage-issues.md:20]:
```suggestion
## Types of Issues and Pull Requests
```

File: docs/src/maintain/manage-issues.md [hunk 2]
```diff
@@ -17,53 +17,54 @@ New issues are filed frequently, and how we respond to those issues directly aff
...
+* **Implementing**: There is an open pull request for each of these issues or this is a pull request that has been approved
 * **Completed**: The issue has been closed (either via pull request merge or by the team manually closing the issue)
 
 We make every attempt to automate movement between as many columns as we can, but sometimes moving issues needs to be done manually.
 
-### When an Issue is Opened
+### When an Issue or Pull Request is Opened
 
-When an issue is opened, it is automatically added to the "Needs Triage" column in the Triage project. These issues need to be evaluated to determine next steps. Anyone on the support team or dev team can follow these steps to properly triage issues.
+When an issue or pull request is opened, it is automatically added to the "Needs Triage" column in the Triage project. These issues and pull requests need to be evaluated to determine next steps. Anyone on the support team or dev team can follow these steps to properly triage issues.
```
Comment (snitin315) [inline docs/src/maintain/manage-issues.md:52]:
```suggestion
When an issue or pull request is opened, it is automatically added to the "Needs Triage" column in the Triage project. These issues and pull requests need to be evaluated to determine the next steps. Anyone on the support team or dev team can follow these steps to properly triage issues.
```

**Review by snitin315 — APPROVED** [review]
There are a few suggestions above. Other than that, LGTM

**Comment by nzakas** [comment]
Ah, it looks like the linting error is actually the link checker. Maybe we should run that as a separate CI job to avoid confusion?

**Comment by snitin315** [comment]
> Ah, it looks like the linting error is actually the link checker. Maybe we should run that as a separate CI job to avoid confusion?

I think it's fine, we anyway need to install deps and build the website to run the link checker.

**Review by snitin315 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**